src/test/java/org/apache/commons/lang3/StringUtilsTest.java

```diff
@@ -1355,15 +1372,6 @@ public void testJoin_Objectarray() {
         assertEquals("foo2", StringUtils.join(MIXED_TYPE_LIST));
     }
 
-    @Disabled
```

Comment:
Was this test removed because the code is already covered elsewhere? Did you confirm this by looking at the JaCoCo report?

Reply:
This is exactly the test that exposes this bug: ```testLang1593```. 
Please find the same assertion here ```testJoin_ArrayOfInt``` in ```StringUtilsTest```

``` assertEquals("1,2", StringUtils.join(INT_PRIM_LIST, SEPARATOR));```

When it comes to coverage, it looks like ```join``` methods have 100% coverage.